apps/web/modules/dashboard/ui/components/conversations-panel.tsx

```
@@ -0,0 +1,214 @@
+import { getCountryFlagUrl, getCountryFromTimezone } from "@/lib/country-utils";
+import { cn } from "@workspace/ui/lib/utils";
+import { DicebearAvatar } from "@workspace/ui/components/dicebear-avatar";
+import { ConversationStatusIcon } from "@workspace/ui/components/conversation-status-icon";
+import { statusFilterAtom } from "@/modules/dashboard/atoms";
+import { useInfiniteScroll } from "@workspace/ui/hooks/use-infinite-scroll";
+import { InfiniteScrollTrigger } from "@workspace/ui/components/infinite-scroll-trigger";
+import { Skeleton } from "@workspace/ui/components/skeleton";
+
+interface ConversationsPanelProps {}
+
+export const ConversationsPanel = ({}: ConversationsPanelProps) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)